### DISCUSSION TITLE
Partitioned tables

### DISCUSSION THREAD

**Discussion by alinselicean** [trimble-oss/dba-dash#596] [Ideas]
Hi 

I recently had an issue with some partitioned tables where the partition maintenance process didn't run for more than 14 days (long story). As a result, new data eventually ended up in the NULL partition (or overflow partition, as I call it). is there a way of monitoring free partitions for partitioned tables, in a similar way IDENTITY values are monitored? 

Thanks,
Alin

**Comment by DavidWiseman** [comment]
Did you manage to fix the issue?  If you changed the data compression on any of the indexes this will cause a problem if the compression doesn't match the staging table.  It should log that the maintenance operation failed but there is probably more I can do to make the issue visible.

**Reply by alinselicean** [reply]
Hi David

I guess I wasn't very clear in my description, my bad :). The partitioned table was on a user database, not related to the DBADash tables. 

Sorry for the confusion.
